### PR TITLE
Make AirPublicInputs fields public

### PR DESCRIPTION
**Pull request by pefontana**
# Make AirPublicInputs fields public

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
  - [ ] CHANGELOG has been updated.

### CODE REVIEW

File: vm/src/air_public_input.rs [hunk 1]
```diff
@@ -56,14 +56,14 @@ impl From<(usize, usize)> for MemorySegmentAddresses {
...
-    public_memory: Vec<PublicMemoryEntry>,
+    pub layout: &'a str,
+    pub rc_min: isize,
+    pub rc_max: isize,
+    pub n_steps: usize,
+    pub memory_segments: HashMap<&'a str, MemorySegmentAddresses>,
+    pub public_memory: Vec<PublicMemoryEntry>,
     #[serde(rename = "dynamic_params")]
-    layout_params: Option<&'a CairoLayout>,
+    pub layout_params: Option<&'a CairoLayout>,
```
Comment (entropidelic) [inline vm/src/air_public_input.rs:66]: from the prover side, we don't really need the layout params as public, at least for the moment. With the name which appears in the `layout` field should be ok 👍

**Review by entropidelic — APPROVED** [review]

File: vm/src/air_public_input.rs (hunk 1 above)
Comment (pefontana) [reply vm/src/air_public_input.rs]: great!
Done [509bed0](https://github.com/lambdaclass/cairo-vm/pull/1474/commits/509bed067ef718e2fa622b71f68351446f17ee5a)

**Comment by Oppen** [comment]
Issues:
– There seems to be a version conflict for the toolchain in the IAI benchmarks
– The PR is labeled `pipelines` rather than having its changelog updated, we should do the latter

**Review by juanbono — APPROVED** [review]

**Comment by pefontana** [comment]
> Issues: – There seems to be a version conflict for the toolchain in the IAI benchmarks – The PR is labeled `pipelines` rather than having its changelog updated, we should do the latter

Yes, you are right @Oppen !
Changelog updated, 
About the CI error, I re-run it and it pass ok

**Comment by Oppen** [comment]
> About the CI error, I re-run it and it pass ok

It still shows as failing https://github.com/lambdaclass/cairo-vm/actions/runs/6851613543/job/18628280029?pr=1474

**Pull request merged**